### DISCUSSION TITLE
Docker Server Running Fine. Patched Snap Camera. NO LENSES SHOWING

### DISCUSSION THREAD

**Discussion by MiX74P3** [ptrumpis/snap-camera-server#164] [Q&A]
OS: Windows 11
Server Version: 3.2.1
Snap Camera Version: 1.2.1
Signature Patch applied: yes

I have docker snap server running fine from what I can tell.
https://i.imgur.com/xmriwgq.png , 
https://i.imgur.com/Mq0CkIh.png
 
Patched Snap Camera
https://github.com/ptrumpis/snap-camera-server/discussions/6
https://ptrumpis.github.io/snap-camera-signature-patch/

No Lenses ever show and none are usable.
https://i.imgur.com/Z1v2BBQ.png

I used this as a standalone awhile back on windows with no issue, so when I saw a server side was an option I went for it. Am I just missing something? I tried looking for help in other threads fist but found no solution. No much on the Windows Side. Thank everyone
 
And where do I locate the log file?

**Comment by 3xoio** [comment]
I dont think this is supported anymore, got the same issue.

**Reply by LunaHackz** [reply]
> I dont think this is supported anymore, got the same issue.

Check your firewall and make sure it's not being blocked!

**Comment by LunaHackz** [comment]
Mine doesn't even show anything at all
https://imgur.com/a/Xzb6nCC

I've spent so much time trying to get everything set up right because the ports in the files didn't work right for me for some reason.  

I can access http://studio-app.snapchat.com/ on port 80 and see my server files (and also at 127.0.0.1:80)
https://imgur.com/a/Ct4K29p

https://app.snapchat.com/  gives me a 404 error (Not Found) along with any port attached to it. (Not sure if that matters)

I can access the db at 127.0.0.1:8080 & http://studio-app.snapchat.com:8080
https://imgur.com/ucSsUip

I've patched the .exe and I've done the unofficial "update" where you transfer over the files from the Lens Studio program to Snap Camera's resources.

The latest log entries have the following:
"Failed to receive scheduled lenses: Connection refused"
"Failed to receive categories: Connection refused"
"Failed to receive community lenses: Connection refused"
"Can't get latest Virtual Camera verison: Connection refused"
"Failed to receive lenses info: Connection refused"
"Failed to receive lenses info: Connection refused"

Refreshing lenses...
"Failed to receive scheduled lenses: Connection refused"
"Failed to receive categories: Connection refused"
"Failed to receive community lenses: Connection refused"

I don't know much about this stuff but I wonder if maybe the server we are supposed to be getting files from is not any good anymore?

**Reply by MiX74P3** [reply]
yea i can access that page as well. just snapcam never loads filters

**Reply by LunaHackz** [reply]
I was able to get the lenses to work. The previews don't show but I can click on each of them and they populate. I don't know what step made it work but I did go through and update the files that are in the repo with the ones that had been altered by others.

**Comment by remenraj** [comment]
Can you check if the host file has `127.0.0.1       studio-app.snapchat.com` added?
